lib/capsulecrm/send_signup_to_capsule.rb

```diff
@@ -15,19 +15,23 @@ class SendSignupToCapsule
   #              'size'            => the organization's size
   #              'sector'          => the organization's sector
   #
-  # organization - a hash containing details of the organization
-  #              'name'           => the org name in Xero - should be the same as that in capsule
+  # party    - a hash containing details of the organization or person
+  #              'name'           => the org / person name in Xero - should be the same as that in capsule
   #              'company_number' => the company number for the organization
   #
   # Returns nil.
-  def self.perform(organization, membership)
-    org = find_organization(organization['name'])
-    if org.nil?
-      Resque.enqueue_in(1.hour, SendSignupToCapsule, organization, membership)
+  def self.perform(party, membership)
+    if membership['product_name'] == "individual"
+      p = find_person(party['name'])
```

Comment:
slightly worried by this; after all, people have overlapping names a lot more than organisations. Can we use email perhaps?